lib/appium.js

```diff
@@ -58,6 +59,9 @@ class AppiumDriver extends BaseDriver {
       } else if (caps.automationName.toLowerCase() === 'youiengine') {
         // but if we do and it is 'YouiEngine', act on it
         return YouiEngineDriver;
+      } else if (caps.automationName.toLowerCase() === 'espresso') {
+        log.warn('The Appium Espresso Driver is currently in beta and meant for experimental usage. Do not use in production.');
```

Comment:
Since people don't typically use Appium "in production" in a meaningful sense anyway, maybe:

> The Appium Espresso driver is currently in early beta and meant only for experimental usage. Its API is not yet complete or guaranteed to work. Please report bugs to the Appium team on GitHub.

Or something?